core/src/test/scala/unit/kafka/integration/BaseTopicMetadataTest.scala

```diff
@@ -201,7 +201,7 @@ abstract class BaseTopicMetadataTest extends ZooKeeperTestHarness {
                                 2000, 0)
         metadata.topicsMetadata.nonEmpty &&
           metadata.topicsMetadata.head.partitionsMetadata.nonEmpty &&
-          expectedIsr == metadata.topicsMetadata.head.partitionsMetadata.head.isr
+          expectedIsr.sorted == metadata.topicsMetadata.head.partitionsMetadata.head.isr.sorted
```

Comment:
I was going to back out the last commit and re-submit with a new modified commit, is that ok?